tests/test_dunders.py

```diff
@@ -453,6 +454,89 @@ def __hash__(self):
         assert 2 == uncached_instance.hash_counter.times_hash_called
         assert 1 == cached_instance.hash_counter.times_hash_called
 
+    def test_cache_hash_serialization(self):
+        """
+        Tests that the hash cache is cleared on deserialization to fix
+        https://github.com/python-attrs/attrs/issues/482 .
+        """
+
+        # first, check that our fix didn't break serialization without
```

Comment:
I _love_ the elaborate explanations here because this problem is indeed rather obscure and easy to forget about. But – and I _do_ realize this is a bit petty 😔 – could you use proper capitalization throughout it please? 😇

Reply:
No problem - I've fixed it.